Dockerfile

```diff
@@ -3,49 +3,49 @@
 FROM debian:jessie
 MAINTAINER Open Knowledge
 
-ENV CKAN_HOME /usr/lib/ckan/default
+ENV CKAN_HOME /usr/lib/ckan
+ENV CKAN_VENV $CKAN_HOME/default
 ENV CKAN_CONFIG /etc/ckan/default
 ENV CKAN_STORAGE_PATH /var/lib/ckan
 ENV CKAN_SITE_URL http://localhost:5000
 
+ENV DEBIAN_FRONTEND noninteractive
+
 # Install required packages
-RUN apt-get -q -y update && apt-get -q -y upgrade && DEBIAN_FRONTEND=noninteractive apt-get -q -y install \
-		python-dev \
-        python-pip \
-        python-virtualenv \
-        libpq-dev \
-        git-core \
-	&& apt-get -q clean
+RUN apt-get -q -y update && apt-get -q -y upgrade && apt-get -q -y install \
+    python-dev \
+    python-pip \
+    python-virtualenv \
+    libpq-dev \
+    git-core \
+    postgresql-client \
+    && apt-get -q clean && rm -rf /var/lib/apt/lists/*
+
+# add ckan user which runs all the ckan related stuff
+RUN useradd -r -u 900 -m -c "ckan account" -d $CKAN_HOME -s /bin/false ckan
```

Comment:
Yes, the variable $CKAN_HOME exists, but the useradd command has trouble creating the dir $CKAN_HOME by itself (contradicting man useradd).

```
Step 9/18 : RUN useradd -r -u 900 -m -c "ckan account" -d $CKAN_HOME -s /bin/false ckan
 ---> Running in 4ad1ca329589
useradd: cannot create directory /usr/lib/ckan/default
ERROR: Service 'ckan' failed to build: The command '/bin/sh -c useradd -r -u 900 -m -c "ckan account" -d $CKAN_HOME -s /bin/false ckan' returned a non-zero code: 12
```

When I move the useradd command below the mkdir step, all is well.

https://askubuntu.com/a/374878 says that adduser is preferred over useradd.